monai/apps/pathology/transforms/spatial/array.py

```diff
@@ -113,14 +113,16 @@ def __init__(
     ):
         self.tile_count = tile_count
         self.tile_size = tile_size
-        self.step = step
         self.random_offset = random_offset
         self.pad_full = pad_full
         self.background_val = background_val
         self.filter_mode = filter_mode
 
-        if self.step is None:
-            self.step = self.tile_size  # non-overlapping grid
+        if step is None:
```

Comment:
Can be: `self.step = step or self.tile_size`?
Or `self.step = self.tile_size if step is None step`.

Thanks.

Reply:
Thank you @Nic-Ma. The first one would fail if step is zero. For the second one, I like what you are suggesting (because it's one line) but I think it is more readable this way and also we are able to add comments to clarify the action.